src/app/zap-templates/zcl/data-model/chip/service-area-cluster.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023-24 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x0150" />
+    <field name="ListOrder"           mask="0x01" />
+    <field name="SelectWhileRunning"  mask="0x02" />
+  </bitmap>
+
+  <!-- Enum definitions for structures-->
+  <enum name="OperationalStatusEnum" type="enum8">
+    <cluster code="0x0150"/>
+    <item value="0x00" name="Pending"/>
+    <item value="0x01" name="Operating"/>
+    <item value="0x02" name="Skipped"/>
+    <item value="0x03" name="Completed"/>
+  </enum>
+
+  <!-- Data types -->
+  <struct name="LocationInfoStruct">
+    <cluster code="0x0150"/>
+    <item name="HomeLocationInfo"  type="HomeLocationStruct"    optional="false" isNullable="true"/>
```

Comment:
This does not match the spec as far as I can tell.

Reply:
Can you expand on how this, and the rest, do not match the spec? What version of the spec are you comparing against?

Reply:
I am comparing against spec rev b3652909bdf595c7634cea1da0f1e58f837ea35f (current tip of master there).

For this specific one, the name does not match the spec, right?  Spec has the field named "LocationInfo".

For the other ones, it's similar things: mismatched names, things that are optional marked mandatory, etc.

Please compare against the spec carefully.

Reply:
You are right. I missed the name. It used to be called that so my brain must have been overlooking it.